projects/orquestra-sdk/pyproject.toml

```
@@ -72,10 +72,16 @@ dependencies = [
 
 [project.optional-dependencies]
 ray = [
-    "ray[default]==2.9.0",
+    "ray==2.9.0",
+    "virtualenv==20.21.0",
```

Is there any reason for the hard pin? [Ray has this requirement open-ended](https://github.com/ray-project/ray/blob/61d9049628001ec6b2129c30d66eb741908994b3/python/setup.py#L252).

```suggestion
    "virtualenv>=20.0.24, !=20.21.1",  # Ray depends on this, but it's missing from the base requirements.
```